samples/nrf_rpc/protocols_serialization/client/README.rst

```diff
@@ -0,0 +1,69 @@
+.. _nrf_rpc_protocols_serialization_client:
+
+Protocols serialization: client
+###############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Protocols serialization client sample demonstrates how to send remote procedure calls (RPC) to a protocols serialization server device, such as one running the :ref:`Protocols serialization server <nrf_rpc_protocols_serialization_server>` sample.
+The RPCs are used to control Bluetooth® LE and :ref:`OpenThread <ug_thread_intro>` stacks running on the server device.
+The client and server devices use the :ref:`nrfxlib:nrf_rpc` and the :ref:`nrf_rpc_uart` to communicate with each other.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-sample-yaml::
+
+To test the sample, you also need another device running the :ref:`Protocols serialization server <nrf_rpc_protocols_serialization_server>` sample.
+
+For testing the Bluetooth LE API serialization, you also need to have the `nRF Connect for Mobile`_ app installed on your smartphone or tablet.
+
+Overview
+********
+
+The Protocols serialization client sample is a thin client that neither includes OpenThread nor Bluetooth LE stacks. Instead, it provides
+implementations of selected OpenThread and Bluetooth LE functions that forward the function calls over UART to the Protocols serialization server.
+The client also includes a shell for testing the serialization.
+
+Configuration
+*************
+
+Snippets
+========
+
+* set the :makevar:`SNIPPET` option to ``ble;coex;debug;log_rpc;openthread``
```

Comment:
Correct me if I'm wrong, @greg-fer, but I don't think this line is needed.

Reply:
Correct.